src/slash_commands/score.rs

```diff
@@ -88,10 +106,11 @@ impl SlashCommand for ScoreCommand {
             _ => Err(format!("Invalid mode: {mode}")),
         }?;
 
-        let han = Han::new(u32::try_from(han)?);
-        let fu = Fu::new(u32::try_from(fu)?);
-        let points = Points::from_calculated(points_calculation_mode, han, fu)?;
-        let embed = create_points_embed(han, fu, &points)?;
+        let han = Han::new(i32::try_from(han)?);
+        let fu = Fu::new(i32::try_from(fu)?);
+        let honbas = Honbas::new(i32::try_from(honbas)?);
```

Comment:
negative honbas 
![image](https://github.com/riichi/chombot/assets/52832022/18b81b18-4dfa-4cbd-a78e-9e42e1895d82)

Reply:
I also thought negative han doesn't make sense, then these Saki card players come and say "wololo we had to manually create the score table".

This is future proof (and not allowed in the default calculation mode anyway).

Reply:
We are making this work for the future generations of chombers to come. We are creating legacy, friend.

Reply:
I am so proud to be a part of this community